src/scss/mixins/_radio-checkbox.scss

```diff
@@ -133,12 +133,95 @@
 /// @access private
 /// @output Styles to modify checkbox/radio labels.
 @mixin _oFormsRadioCheckboxLabel {
-	@include oTypographySize($_o-forms-typography-scale-regular);
-	display: inline-block;
-	padding-left: $_o-forms-radiocheckbox-default-size + $_o-forms-label-padding-left; // radio/checkboxes are being styled as pseudo-elements, which means we have to account for the width of the radio/checkboxes when adding padding to the labels
-	position: relative;
-	cursor: pointer;
-	font-weight: oFontsWeight('normal');
-	padding-top: 2px; // To align the text with the center of the checkbox/radio
-	user-select: none;
+  @include oTypographySize($_o-forms-typography-scale-regular);
+  display: inline-block;
+  padding-left: $_o-forms-radiocheckbox-default-size + $_o-forms-label-padding-left; // radio/checkboxes are being styled as pseudo-elements, which means we have to account for the width of the radio/checkboxes when adding padding to the labels
+  position: relative;
+  cursor: pointer;
+  font-weight: oFontsWeight('normal');
+  padding-top: 2px; // To align the text with the center of the checkbox/radio
+  user-select: none;
+}
+
+@mixin oFormsRadioButtonsStyled($selectors: (legend: '.o-forms__radio-buttons__legend', wrapper: '.o-forms__radio-buttons__wrapper', label: '.o-forms__label', radio: '.o-forms__radio-buttons__button')) {
```

Comment:
In v5 we removed the ability to rename these selectors with mixin parameters as it was implemented inconsistently.
https://github.com/Financial-Times/o-forms/blob/ca76910f71c5aadf599c0ed4c05d441fc5c6582f/src/scss/mixins/_radio-checkbox.scss#L44